examples/manager.py

```diff
@@ -0,0 +1,24 @@
+'''
+Waits for a keyboard interupt, and sends a kill signal to the other nodes
+'''
+
+import time
+import zmq
+
+
+def main():
+    context = zmq.Context()
+    shutdown = context.socket(zmq.PUB)
+    shutdown.bind("tcp://*:9000")
+
+    shutdown.send(b'START')
+    try:
+        while True:
+            time.sleep(1)
+    except KeyboardInterrupt:
+        print("Sending kill signal")
+        shutdown.send(b"EXIT")
```

Comment:
Yeah, that sounds like a good idea. I'm not 100% sure if we should tell process to exit like this, so I didn't put too much effort into it. If a process manager just requests processes to stop, I can see that request being accidentaly ignored or missed. Might be better to have a more agressive way of bringing down worker processes before they run out of control.